src/sbt-test/sbt-bundle/basic/build.sbt

```diff
@@ -53,3 +55,15 @@ checkBundleConf := {
                             |}""".stripMargin
   contents should include(expectedContents)
 }
+
+val checkConfigDist = taskKey[Unit]("check-config-dist-contents")
```

Comment:
We should have a separate scripted test for testing config - perhaps leave the existing test as-is.